### PR TITLE
Bump bevy to 0.12

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

**Comment by TristanCacqueray** [comment]
This is blocked by https://github.com/aevyrie/bevy_mod_picking/pull/276 and a new version of cuicui.

File: Readme.md [hunk 1]
```diff
@@ -629,6 +629,7 @@ fn return_trigger_action(mut requests: EventWriter<NavRequest>, input: Res<Input
 
 | bevy | latest supporting version      |
 |------|--------|
+| 0.11 | 0.33.0 |
```
Comment (AlexAegis) [inline Readme.md]: typo?

Comment (TristanCacqueray) [reply Readme.md]: yes, thanks.

**Comment by nicopap** [comment]
`bevy_mod_picking` is ready. If you manage to make the tests pass, I can merge this.

**Comment by nicopap** [comment]
I'm taking care of this, no need to continue work on this branch

**Pull request closed without merging**